SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -93,6 +107,33 @@ void UGlobalStateManager::ApplyAcceptingPlayersUpdate(bool bAcceptingPlayersUpda
 	}
 }
 
+void UGlobalStateManager::SendShutdownMultiProcessRequest(bool bValue)
```

Comment:
I'm more inclined to call this label this function based on the event it's responding to. OnPrePIEEnded. What's bValue used for?

Reply:
Agreed. 

bValue isn't used for anything but the parameter is required since the delegate takes in a boolean when broadcasting the event.